mysql_type/__init__.py

```diff
@@ -96,6 +96,8 @@ def replace_arg(mo) -> str:
                     raise Exception("Number of _LIST_ arguments do not match")
                 if isinstance(a, list):
                     flatargs += a
+                    if not a:
+                        raise Exception("Value of a _LIST_ argument was empty")
```

Comment:
Instead of throwing an exception we could `return "null"`.

This should convert the code to `foo in (null)` which evaluates to false.